src/services/variables.ts

```diff
@@ -16,8 +16,8 @@ export const VAR_FIELDS = 'fields';
 export const VAR_FIELDS_EXPR = '${fields}';
 export const VAR_PATTERNS = 'patterns';
 export const VAR_PATTERNS_EXPR = '${patterns}';
-export const VAR_LEVELS = 'levels';
```

Comment:
We support levels plural, what was the reason behind this change?

Reply:
because the actual name of the label is `level`, when calling `label/level/values`

Reply:
We could revert the change and hardcode the 'level' string in the `getTagValuesProvider` though